### PR TITLE
remove 400 - Bad Request FAQ

### PR DESCRIPTION
**Pull request by maryamsulemani97**
part of #1593 

Moves content from the `400 - Bad Request` FAQ to Core concepts/Documents

### CODE REVIEW

**Review by dichotommy — COMMENTED** [review]
I have mixed feelings about moving this content to the documents guide. On the one hand, the guide is already a bit messy so it doesn't make things significantly worse. On the other hand, this info seems mostly useful for troubleshooting; it's not something essential that every user needs to know (and it's not conceptual).

I think it would make more sense in the API reference, under the add documents endpoints, or in the errors page, under the `malformed_payload` code. I also think it could be shortened a lot: instead of providing an example, we could just say make sure that all documents are in an array and that the JSON is correctly formatted. But perhaps I'm overthinking this?

@guimachiavelli WDYT?

**Comment by guimachiavelli** [comment]
I think @dichotommy's suggestion that this is probably longer than it should is relevant. That FAQ answer also focusses too much on commas, when malformed JSON occurs because of many other things (missing quotes around a key, mismatched [] and {}, etc). If we simplify the text and remove the examples and the tip, it could easily fit into either Core Concepts or the Errors page. It could even be adequate to mention it in both pages, actually, if we're ok with minor content duplication.

Out of scope, but it is interesting that you mention troubleshooting, as that is something I was reading about on _Docs for Devs_. To keep things short, the authors recommend having a troubleshooting reference with instructions on how to fix common errors. Not 100% how that would work, especially from an IA perspective, but could be interesting to discuss.

File: resources/faq.md [hunk 1]
```diff
@@ -67,44 +67,9 @@ This response indicates that the operation has been taken into account and will
...
-
-Wrong:
-
-```json
-{
-  "id": 123,
-  "title": "Pride and Prejudice"
-}
-```
+The `400 - Bad request` response probably means that your data is not in an expected format. You might have extraneous commas, mismatched brackets, missing quotes,etc.. Meilisearch API accepts JSON, CSV, and NDJSON formats.
```
Comment (maryamsulemani97) [inline resources/faq.md:70]:
```suggestion
The `400 - Bad request` response probably means that your data is not in an expected format. You might have extraneous commas, mismatched brackets, missing quotes, etc. Meilisearch API accepts JSON, CSV, and NDJSON formats.
```

**Review by guimachiavelli — DISMISSED** [review]
Two suggestions, one very minor, and we should be good to go

**Review by guimachiavelli — APPROVED** [review]
🐤

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**